.changes/1006.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "added",
+    "description": "add CentOS7-compatible target for aarch64"
```

Comment:
Can you add a line with `"issues": [528]`, since I've just noticed this also will close #528. So the end result is:

```json
{
    "type": "added",
    "description": "add CentOS7-compatible target for aarch64",
    "issues": [528]
}